src/monika-config-schema.json

```diff
@@ -1141,6 +1141,38 @@
                 }
               }
             }
+          },
+          {
+            "title": "Instatus",
+            "type": "object",
+            "required": ["id", "type", "data"],
+            "additionalProperties": false,
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "Notification identity number",
```

Comment:
"The type of notification" instead of "Notification identity number"

Reply:
ok, description is fixed